ndc-test/src/test_cases/query/common.rs

```diff
@@ -35,6 +46,7 @@ pub fn select_columns(
                 models::Field::Column {
                     column: f.0.clone(),
                     fields: None,
+                    arguments: None,
```

Comment:
Now if the fields are nullable, we still need to provide them. It's probably easier to just exclude even fields with nullable arguments for now.

Reply:
Aren't we doing this by having the arguments map optional? Otherwise I could just make the arguments map required regardless of what the arguments are.

Reply:
Until we make (all) nullable inputs optional, for now, the individual arguments need to be provided as null, so this technically is an invalid request. Later we can just omit anything which is nullable.

Reply:
Ok I've set arguments to null if they aren't required.

Reply:
`arguments` is still set to `None`? We need to either set it to a map with one entry for each argument, or we need to change the `matches!` macro to exclude any field with arguments (until the follow up PR).

Reply:
7036c49..ec562f5